Kernel/Storage/StorageManagement.cpp

```diff
@@ -111,7 +111,8 @@ UNMAP_AFTER_INIT void StorageManagement::enumerate_pci_controllers(bool force_pi
 
             if (subclass_code == SubclassID::SATAController
                 && device_identifier.prog_if().value() == to_underlying(PCI::MassStorage::SATAProgIF::AHCI)) {
-                m_controllers.append(AHCIController::initialize(device_identifier));
+                if (auto ahci_controller_or_error = AHCIController::initialize(device_identifier); !ahci_controller_or_error.is_error())
```

Comment:
are we interested in this error or not really? (for debugging purposes?)

Reply:
Right, I forgot to add that line. The other controllers throw error messages into the debug log as well.